impl_core/src/main/java/io/opencensus/implcore/trace/propagation/B3Format.java

```diff
@@ -58,7 +62,8 @@
   }
 
   @Override
-  public <C> void inject(SpanContext spanContext, C carrier, Setter<C> setter) {
+  public <C /*>>> extends @NonNull Object*/> void inject(
+      SpanContext spanContext, C carrier, Setter<C> setter) {
     checkNotNull(spanContext, "spanContext");
```

Comment:
Should we also remove guava here an use our own implementation?

Reply:
Same here, I think that should be a separate change.